cucumber-messages/dotnet/messages.proto

```diff
@@ -459,26 +483,26 @@ message TestStepMatched {
 }
 
 message TestStepStarted {
-  reserved 1;
-  uint32 index = 2;
+  reserved 1, 2, 4;
   Timestamp timestamp = 3;
-  string testCaseId = 4;
+  string testStepId = 5;
```

Comment:
I think its probably useful to have both the testCaseId and testStepId. Given the TestCase message you defined, I expect we would keep the testStepIds under a testCase object